### DISCUSSION TITLE
What is the name of a cocktail made with vodka, Kahlua/Tia Maria, and Coke?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#774] [Galaxy Trivia] [answered]
**Bonus:** Describe the steps to make it.

**Comment by peterbaccaro** [comment, marked as answer]
The name of the cocktail made with vodka, Kahlua/Tia Maria, and Coke is called a **Black Russian**. 

Here are the steps to make it:

**Ingredients:**

- 2 oz of Vodka
- 1 oz of Kahlua or Tia Maria
- 3 oz of Coca-Cola
- Ice
- Glassware: highball glass or rocks glass

**Directions:**

- Fill a glass with ice.
- Add the vodka and Kahlua or Tia Maria into the glass.
- Stir the mixture until the ingredients are combined.
- Slowly pour the Coca-Cola into the glass.
- Stir the mixture gently to mix the Coke with the other ingredients.
- Garnish with a lime wedge, if desired.

The resulting drink should be dark in colour, with a slight fizz from the Coca-Cola, and a sweet, slightly bitter flavour from the Kahlua/Tia - Maria. It's a simple yet delicious cocktail that is perfect for those who enjoy a sweeter, richer drink.

**Comment by SANKARAMDAS** [comment]
The Black Russian drink supposedly first appeared in Brussels in 1949. It's a wonderfully simple cocktail, but you wouldn't know that from the flavor. This drink doesn't really have anything to do with Russia, other than the fact that it contains vodka. The main flavor in this drink is simply the Kahlua.

The Black Russian cocktail is a drink made from vodka and coffee liqueur can be served on the rocks (with ice only) but when combined with some cola it's much more palatable.
The black Russian is less famous than it's counterpart the white russian, but is still just as enjoyable.
The measures given are as close as realistically possible to the official IBA guidelines, which is 5 parts vodka to 2 parts coffee liqueur.

Equipment :
Lowball glass

Ingredients :
62 ml vodka
25 ml coffee liqueur, like Kahlua or Tia Maria 
cola
ice cubes
To garnish
a wedge of lime

Instructions:
Add several ice cubes to a short glass. 
Pour in the vodka measure over the ice cubes.
Then pour in the coffee liqueur measure.
Add the cola and stir again, then your ready to enjoy.
Optional – garnish the glass with a wedge of lime

**Comment by ParanoidUser** [comment]
**Extra:** Let's request an AI-generated image of the cocktail. Shall we? 😄

**Reply by peterbaccaro** [reply]
![image](https://github.com/ParanoidUser/yolo/assets/74666079/5b45b5d9-3f2b-4173-95ae-2921ddf0c0fc)

**Reply by SANKARAMDAS** [reply]
![360_F_560209394_t60RabkH6in0pJBtiLuGJZje9u5aHysA](https://github.com/ParanoidUser/yolo/assets/31897843/b5a378a1-ff62-4523-ba3f-8a9708f95693)

**Reply by ParanoidUser** [reply]
"Phew... 😮‍💨 *exhaled all bartenders around the world* as long as AI believes it's a 'Black Russian' 🍹 our jobs are secure!"
